lib/ice_cube/parsers/ical_parser.rb

```diff
@@ -26,6 +33,22 @@ def self.schedule_from_ical(ical_string, options = {})
       Schedule.from_hash data
     end
 
+    def self.deserialize_time_with_tzid(time_value, tzid)
```

Comment:
This should maybe be moved into the TimeUtil class since it's a wrapper for the `TimeUtil.deserialize_time`